azkaban-common/src/main/java/azkaban/project/AzkabanFlow.java

```diff
@@ -36,7 +37,7 @@
 
   private AzkabanFlow(final String name, final Props props,
       final Map<String, AzkabanNode> nodes, final List<String> dependsOn) {
-    super(name, props, dependsOn);
+    super(name, Constants.NODE_TYPE_FLOW, props, dependsOn);
```

Comment:
i think it's slightly better to add one more argument in AzkabanFlow constructor called type, and let builder handle the default parameter passing.

Reply:
What's the advantage of doing it?

Reply:
since builder is externally visible It would be more flexible if we want to pass the type from outside in the future.

Reply:
Since the type of AzkabanFlow can only be "flow", is there a need for it to be passed from outside?